src/components/loginPage.jsx

```diff
@@ -54,12 +45,12 @@ const LoginPage = () => {
           <input
             type="email"
             name="email"
-            value={email}
+            value={loginDetails?.email}
```

Comment:
Optional chaining is not necessary here... remove '?'
```suggestion
            value={loginDetails.email}
```